components/single_image_view.jsx

```diff
@@ -190,13 +186,15 @@ export default class SingleImageView extends React.PureComponent {
                 >
                     {fileHeader}
                     <div
-                        style={imageContainerDimension}
                         className='image-container'
                     >
-                        <div className={`image-loaded ${fadeInClass}`}>
+                        <div
+                            className={`image-loaded ${fadeInClass}`}
+                            style={imageContainerStyle}
```

Comment:
Move the height setting down one level and set it always to ensure that height will not change